packages/flutter_tools/test/commands.shard/permeable/upgrade_test.dart

```diff
@@ -282,6 +282,43 @@ void main() {
       Platform: () => fakePlatform,
     });
 
+    testUsingContext('Show current version to the upgrade message.', () async {
+      const String revision = 'abc123';
+      const String upstreamRevision = 'def456';
+      const String version = '1.2.3';
+      const String upstreamVersion = '4.5.6';
+
+      when(flutterVersion.frameworkRevision).thenReturn(revision);
+      when(flutterVersion.frameworkRevisionShort).thenReturn(revision);
```

Comment:
I don't think you'll need `flutterVersion.frameworkRevisionShort` in your test.
Same for `latestVersion.frameworkRevisionShort` below.